src/lib/ifaces/mac_vtap.rs

```diff
@@ -5,8 +5,8 @@ use crate::NisporError;
 use netlink_packet_route::rtnl::link::nlas;
 use serde_derive::{Deserialize, Serialize};
 
-#[serde(rename_all = "lowercase")]
 #[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+#[serde(rename_all = "lowercase")]
```

Comment:
I think this change is unrelated, is because of rustfmt? Does it matter now?

Reply:
The cargo clippy warn me that `serde` is used before derive introduced it.

Reply:
Moved to https://github.com/nispor/nispor/pull/119